swagger.yaml

```diff
@@ -1622,6 +1622,31 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /day2_clusters:
```

Comment:
Why create a new path instead of using the existing register-cluster with a new parameter?
Also don't use "day2" in user-facing code.  Maybe "previously-installed" or something.

Reply:
we want a different path in order to keep a day2 code as separate as possible and not to interfere with day1. Especially since this solution is only temporary until we get assisted-service running on OCP cluster. 
I would need a more precise name